### DISCUSSION TITLE
How to Model EEG Data and Ratings Using ANN in MATLAB?

### DISCUSSION THREAD

**Discussion by wrkndrvmbhnhfzo** [community/community#148210] [Programming Help]
### Body

I have cleaned EEG signal data in .mat files collected from 20 individuals while listening to 12 songs, and each participant rated their familiarity with and liking of the songs on a scale of 1-5. These ratings are stored in an Excel file. I want to model this data using an artificial neural network (ANN) in MATLAB. I've been told that I don't need to use EEGLAB since the data is already cleaned, but I have 12*20=240 .mat files, each representing separate matrices, and the files are large.

Could you guide me on how to approach this step-by-step? Your assistance would be greatly appreciated

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jakubbbdev2** [comment]
Hi @wrkndrvmbhnhfzo can you add me on Discord? I

**Reply by wrkndrvmbhnhfzo** [reply]
my username is the same its wrkndrvmbhnfzo you can add me

**Reply by jakubbbdev2** [reply]
its not working add me: hobbysigma
